README.md

```diff
@@ -185,6 +185,15 @@ To run new frontend (React) tests type:
 ```
 npm test
 ```
+##### 13. React documentation
+Start a style guide dev server:
```

Comment:
@jmiranda do you think you would like to keep this info here in general README or some place different?

Reply:
README seems fine for now since the README is geared towards developers. I'll add some documentation to the readthedocs content later.